version.py

```diff
@@ -48,4 +48,4 @@
 #   historical commits sharing the same major version, in which case a
 #   minor version bump suffices.
 
-PROVISION_VERSION = (280, 0)  # last bumped 2024-05-31 for upgrading Python requirements
+PROVISION_VERSION = (280, 1)  # last bumped 2024-05-31 for upgrading Python requirements
```

Comment:
Just a casual note: you should update the comments too (though now you might have to do both to resolve merge conflicts)